### PR TITLE
Announce 4.94

### PR DESCRIPTION
**Pull request by fredemmott**

### CODE REVIEW

**Review by lexidor — CHANGES_REQUESTED** [review]
`enable_enum_supertyping` was promoted from an unstable feature to something that can be enabled in hhconfig. The error message you get from `enum X: int { use Y; }` suggests the hhconfig flag.
```
Typing[4408] This Enum uses syntax reserved for the Enum Supertyping feature.
Enable it with the enable_enum_supertyping option in .hhconfigHack(4408)
```

https://github.com/facebook/hhvm/commit/b509e5488870cddd731b077db54037f2c18d91d6#diff-694d0457514794183eda766798efc3508d16bc9fd19b75c9c8e88204f51ef159

**Comment by fredemmott** [comment]
In general, features that need an opt-in aren't considered ready; they can potentially be used in monorepos or top-level projects, but not at any other level without splitting the ecosystem. There's also not yet documentation for this feature.

File: _posts/2021-01-25-hhvm-4.94.markdown [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: "HHVM 4.94.0, 4.88.1, 4.91.1, 4.92.1, and 4.93.1"
+layout: post
+author: fred
+category: blog
+---
+
+HHVM 4.94 is released! This release marks the end of support for 4.86;
+HHVM 4.87&ndash;4.92 remain supported, as do the 4.56 and 4.80 LTS releases.
```
Comment (jjergus) [inline _posts/2021-01-25-hhvm-4.94.markdown:9]:
```suggestion
HHVM 4.87&ndash;4.93 remain supported, as do the 4.56 and 4.80 LTS releases.
```

File: _posts/2021-01-25-hhvm-4.94.markdown [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  than 8GB of RAM; this included most TravisCI and CircleCI configurations.
+- Fixed line numbers for some runtime errors being incorrectly reported as -1.
+- Fixed `debug_zval_dump()` incorrectly reporting a refcount of `1` for
+  uncounted or static values; these are now explicitly marked as `uncounted` or
+  `static`.
+- The error message for attempting to use first class function pointer syntax
+  with instance methods (as opposed to static methods) now suggests
+  `meth_caller()`.
+- first class function pointers are now supported by debugging functions such as
+  `print_r`, `var_dump`, and `=` in hphpd.
```
Comment (jjergus) [inline _posts/2021-01-25-hhvm-4.94.markdown:28]: nit: I think we settled on calling these "function references" oficially; also probably worth linking to https://docs.hhvm.com/hack/functions/function-references when they're first mentioned.

**Review by jjergus — APPROVED** [review]

File: _posts/2021-01-25-hhvm-4.94.markdown (hunk 2 above)
Comment (fredemmott) [reply _posts/2021-01-25-hhvm-4.94.markdown:28]: I think we should probably change the docs here: most recent discussions say FCFP, as do the majority of typechecker error messages - "Invalid function reference. This function requires reified generics. Prefer using a lambda instead. (Typing[4381])" is a difference

**Pull request merged**